### DISCUSSION TITLE
What database(-s) do you use?

### DISCUSSION THREAD

**Discussion by LonwoLonwo** [dbeaver/dbeaver#16186] [Polls]
It would be very interesting for us to know - what database do you use most often when you use DBeaver?
Best regards, DBeaver team.

**Comment by LonwoLonwo** [comment]
Unfortunately, the vote options are offered by GitHub only 8, and it is regrettable because there are many excellent databases in the world.  
But you can always write your answer in the comments.
We are very interested in your opinion.

**Comment by wistlo** [comment]
mySQL
migrating some work to Postgres (large tables)
Oracle (but use TOAD for that.  Once is enough for setting up an Oracle Client connection. )

**Comment by flambchap** [comment]
SQLite and CockroachDB these days.

**Reply by LonwoLonwo** [reply]
Any particular reason to use CockroachDB? I'm curious.

**Reply by flambchap** [reply]
Cockroach Labs provides a free plan for up to 5GB of storage, which is more than enough for my purposes.

**Reply by flambchap** [reply]
https://www.cockroachlabs.com/get-started-cockroachdb/

**Reply by TerryCrawf** [reply]
Can you use dbeaver to connect to CockroachDB cluster?

**Reply by flambchap** [reply]
@TerryCrawf Yes, it works like a charm.

**Reply by TerryCrawf** [reply]
Spent a couple of hours this afternoon and managed to get it connected - works well

**Comment by mkgrgis** [comment]
PostgreSQL, Firebird, SQLite, Oracle 10-12.

**Comment by lzb-libra** [comment]
Recently, I have given up using DBeaver. This software is becoming more and more difficult to use. Every update needs to reinstall the downloaded plug-ins, and the recently updated version can only use jdk11, and none of my suggestions During the installation, various pop-up windows report errors during use, and you need to reconnect. These are some of the problems I encountered during use, because the problems encountered during use have not been dealt with because I gave up using this product. software root cause

**Comment by topvis** [comment]
SQLite

**Comment by RobertHana** [comment]
Snowflake
Postgres
MySQL
H2
SQLServer

**Comment by lzb-libra** [comment]
mysql
SQLite
